charts/quickwit/templates/_helpers.tpl

```diff
@@ -190,20 +200,35 @@ Quickwit metastore environment
 - name: QW_METASTORE_URI
   value: {{ .Values.config.metastore_uri }}
 {{- else if .Values.config.postgres }}
+- name: QW_METASTORE_URI
+  value: "postgres://$(POSTGRES_USERNAME):$(POSTGRES_PASSWORD)@$(POSTGRES_HOST):$(POSTGRES_PORT)/$(POSTGRES_DATABASE)"      
+{{- end }}
```

Comment:
does it work to move the definition of `QW_METASTORE_URI` up here? According to [this doc](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/) the order is important.